astropy/cosmology/flrw.py

```diff
@@ -1353,8 +1354,7 @@ def kpc_comoving_per_arcmin(self, z):
             The distance in comoving kpc corresponding to an arcmin at each
             input redshift.
         """
-        return (self.comoving_transverse_distance(z).to(u.kpc) *
-                arcmin_in_radians / u.arcmin)
+        return self.comoving_transverse_distance(z).to(u.kpc) / radian_in_arcmin
```

Comment:
Is the calculation costly compared to unit conversions? If so, my tendency would be to replace this with
```
...to(u.kpc/u.arcmin, equivalencies=u.dimensionless_angles())
```
I think that makes it a lot clearer what happens. But this unit conversion is a bit slower, so if that matters, sticking with what you have is fine too.

Reply:
For `z` an array, the timing difference is trivial.
For scalar `z`... not so much

Reply:
![Screen Shot 2021-11-02 at 17 37 23](https://user-images.githubusercontent.com/8949649/139955324-fa6314e9-4274-4ae3-b8b6-073c14603a60.png)

Reply:
The other is definitely clearer, but the speed does matter.

Reply:
I should produce a timing profile of cosmology and see where the bottlenecks are. It would be good to make sure I'm not sweating the 1% stuff if there's a 10% speedup to be had!

Reply:
OK, sounds like for now it is best indeed to keep as is!